src/toil_scripts/adam_pipeline/spark_toil_script.py

```diff
@@ -48,7 +50,7 @@ def start_master(job, inputs):
     """
     log.write("master job\n")
```

Comment:
I have a [branch](https://github.com/BD2KGenomics/toil-scripts/compare/master...fnothaft:adam-cleanup?expand=1) that fixes this, but it is out of date.